storage/storage.go

```diff
@@ -290,6 +295,14 @@ type Password struct {
 	UserID string `json:"userID"`
 }
 
+// Connector is an object that contains the metadata about connectors used to login to Dex.
+type Connector struct {
```

Comment:
Can you add comments to the individual fields? E.g. what's an ID, how is it different then name? What are examples of types. What is the config slice expected to hold?